src/migrations/20190822125944-create-verify-user.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('VerifyUsers', {
```

Comment:
We do not need another table to keep records of our verified users.
The boolean `isVerified` which you added in `src/migrations/20190814125500-create-user.js` lines 22-24 should suffice.
Add the properties of the `VerifyUsers` table to the `Users` table.
> A verified user is still a user